sitemedia/js/controllers/search_controller.js

```diff
@@ -36,6 +43,24 @@ export default class extends Controller {
         }
     }
 
+    filterTargetConnected() {
+        // on load, check if filters are active and update label accordingly
+        this.updateFilters();
+    }
+
+    updateFilters() {
+        // toggle active filters label based on checked filter checkboxes
+        if (this.filterTargets.some((target) => target.checked)) {
+            this.filterLabelTargets.forEach((label) =>
+                label.classList.add("active")
+            );
+        } else {
+            this.filterLabelTargets.forEach((label) =>
+                label.classList.remove("active")
+            );
+        }
+    }
```

Comment:
can't this be done server side? we just need to set an `active` class on the filter toggle if there are filters applied, right? how about a simple boolean method on the form that checks if there are filters active 

my inclination is that the we shouldn't display the active status until the filters have been applied to the search; once we implement the reactive search they will be applied automatically, but I think the server side active status still works in that case

Reply:
Good point!